make.js

```diff
@@ -0,0 +1,14 @@
+function make(...args) {
+  let array = [];
+
+  function func(...el) {
+    if (typeof el[0] === 'function') {
```

Comment:
name for spreed function parameters could be more meaningful than `el`, as and function name `func`